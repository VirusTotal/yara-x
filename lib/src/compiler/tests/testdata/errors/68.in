rule test { meta: condition: true }