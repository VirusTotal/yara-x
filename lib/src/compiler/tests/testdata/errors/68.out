error: syntax error
 --> line:1:19
  |
1 | rule test { meta: condition: true }
  |                   ^ expected identifier
  |